proto/bitsong/nft/v1beta1/query.proto

```
@@ -0,0 +1,136 @@
+syntax = "proto3";
+package bitsong.nft.v1beta1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.55.1)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)